crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -238,9 +239,10 @@ async fn symbolicate_js_frame(
             .map(|base| join_paths(base, &filename))
             .unwrap_or_else(|| filename.clone());
 
-        frame.in_app = frame
-            .in_app
-            .or_else(|| is_in_app(&frame.abs_path, &filename));
+        let in_app = is_in_app(&frame.abs_path, &filename);
+        if in_app.is_some() {
+            frame.in_app = in_app;
+        }
```

Comment:
I would even say that we can unconditionally overwrite with `None` as well, as the sentry processor would not merge `None` into the frame. But it might be worth double checking that.

Reply:
Indeed, `_merge_frame` checks for `None` before merging.